Automattic/woocommerce-payments
# Add AVS rule to Fraud & Risk tools

Fixes Automattic/woocommerce-payments-server#3346

#### Changes proposed in this Pull Request

This PR aims to add the AVS rule to the advanced settings page of the Fraud & Risk tools.









#### Testing instructions





* Checkout to this branch
* Listen to Stripe webhooks
* Go to **Payments > Settings**
* Scroll down to the **Fraud protection** section, select "Advanced" and click on "Configure" (or "Edit" if it has already been configured)
* The toggle on the rule cards means that now if it should be blocked, so there is no "Review" option – make sure the UI only mentions the payment being blocked
* Search for the "AVS mismatch" rule and toggle it, then save
* Access [Stripe's dashboard](https://dashboard.stripe.com/test/logs?showIP=false&created[preset]=1H)
  * Check the latest `POST /v1/accounts/YOUR_ACCOUNT_ID`
  * Scroll the right container to the bottom of the page and you should see a similar **Request POST body**

<img width="286" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/16882226/4f9e64db-581d-4fa0-b52d-f6a0b78092b9">


-------------------


**Post merge**
